src/compiler/compile/Component.ts

```diff
@@ -143,8 +145,16 @@ export default class Component {
 					.replace(process.cwd(), '')
 					.replace(regex_leading_directory_separator, '')
 				: compile_options.filename);
-		this.locate = getLocator(this.source, { offsetLine: 1 });
-
+		const locator = getLocator(this.source, { offsetLine: 1 });
+		const tracer = compile_options.sourcemap ? new TraceMap(compile_options.sourcemap) : undefined;
+		this.locate = (c) => {
+			let location:{line:number,column:number} = locator(c);
```

Comment:
it's not currently running, but I expect `lint` will want you to add spaces here
```suggestion
			let location: { line: number, column: number } = locator(c);
```